docker/1.4.8/config/README.md

```diff
@@ -0,0 +1,4 @@
+# Configuration
+
+This directory is mounted as a volume for the docker container running Mosquitto.
+
```

Comment:
I have created different directories for each mosquitto version since the Dockerfile will be different for each version. This also gives flexibility in case other files need to be added for different versions of Mosquitto down the line. In order to have an official image in Docker Hub, you need to have a specific Dockerfile available that will be used to build that specific image (i.e. mosquitto-1.4.8). I am trying to follow the Docker Hub conventions that I am seeing with other images. We could just have 1 latest and greatest image, but I would prefer to allow older Mosquitto versions to be made available as well.